src/domElement.js

```diff
@@ -1,7 +1,7 @@
-const form = document.forms[0];
-const addField = form.elements.todo;
-const returnIcon = document.querySelector('.fa-level-down');
-const clearTodo = document.querySelector('.clear');
+var form = document.forms[0];
+var addField = form.elements.todo;
+var returnIcon = document.querySelector('.fa-level-down');
+var clearTodo = document.querySelector('.clear');
```

Comment:
- [ ] Kindly use the let keyword when defining variables. Variables defined with var inside of a function “lose scope” and are essentially destroyed when the function exits. Follow the correct JS practices.